scripts/hypothesistooling.py

```diff
@@ -120,7 +120,8 @@ def has_source_changes(version=None):
     point_of_divergence = merge_base('HEAD', version)
 
     return subprocess.call([
-        'git', 'diff', '--exit-code', point_of_divergence, 'HEAD', '--', SRC,
+        'git', 'diff', '-s', '--exit-code', point_of_divergence, 'HEAD', '--',
```

Comment:
I’d rather use `--no-patch` than `-s`. It’s clearer for future readers, and characters aren't at a premium.

Reply:
That's a great point, thanks.